src/content/5/ptbr/part5a.md

```diff
@@ -604,50 +618,53 @@ User details of the logged-in user do not have to be saved to the local storage
 }
 ```
 
-#### 5.2: bloglist frontend, step2
+### 5.2: frontend do bloglist, passo 2
+
 
-Make the login 'permanent' by using the local storage. Also, implement a way to log out.
+Torne o login 'permanente' usando o armazenamento local. Além disso, implemente uma maneira de deslogar.
 
-![browser showing logout button after logging in](../../images/5/6e.png)
+![navegador mostrando o botão de logout após o login](../../images/5/6e.png)
 
-Ensure the browser does not remember the details of the user after logging out.
+Assegure-se de que o navegador não lembre os detalhes do usuário após o logout.
 
-#### 5.3: bloglist frontend, step3
+#### 5.3: frontend do bloglist, passo 3
 
-Expand your application to allow a logged-in user to add new blogs:
+Expanda sua aplicação para permitir que um usuário logado adicione novos blogs:
 
-![browser showing new blog form](../../images/5/7e.png)
+![navegador mostrando o formulário do novo blog](../../images/5/7e.png)
 
-#### 5.4: bloglist frontend, step4
+#### 5.4: frontend do bloglist, passo 4
 
-Implement notifications that inform the user about successful and unsuccessful operations at the top of the page. For example, when a new blog is added, the following notification can be shown:
+Implemente notificações que informem o usuário sobre operações bem-sucedidas e mal-sucedidas no topo da página. Por exemplo, quando um novo blog é adicionado, a seguinte notificação pode ser exibida:
 
-![browser showing successful operation](../../images/5/8e.png)
+![navegador mostrando a operação bem-sucedida](../../images/5/8e.png)
 
-Failed login can show the following notification:
+Um login mal-sucedido pode mostrar a seguinte notificação:
 
-![browser showing failed login attempt](../../images/5/9e.png)
+![navegador mostrando a tentativa de login mal-sucedida](../../images/5/9e.png)
 
-The notifications must be visible for a few seconds. It is not compulsory to add colors.
+As notificações devem ser visíveis por alguns segundos. Não é obrigatório adicionar cores.
 
 </div>
 
 <div class="content">
 
-### A note on using local storage
+### Uma nota sobre o uso do armazenamento local
+
+
+No [fim](/en/part4/token_authentication#problems-of-token-based-authentication) da última parte, nós mencionamos que o desafio da autenticação baseada em token é como lidar com a situação em que o acesso da API do titular do token à API precisa ser revogado.
```

Comment:
Também substituir este link "/en/part4/token_authentication#problems-of-token-based-authentication" pela versão ptbr.